### PR TITLE
Html seat booking task 

### PR DESCRIPTION
**Pull request by f3d0t**
**Html seat booking task** 
[Demo](https://f3d0t.github.io/f3d0t/html-css-homework/)
[Code base](https://github.com/f3d0t/f3d0t/tree/main/html-css-homework)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/f3d0t/html_movie_seat_booking/index.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+					<li class="header_menu__item"><a href="#">Ongoing</a></li>
+					<li class="header_menu__item header_menu__item--active"><a href="#">Booking</a></li>
+					<li class="header_menu__item"><a href="#">Bestsellers</a></li>
+					<li class="header_menu__item"><a href="#">Film rating</a></li>
+				</ul>
+			</nav>
+		</header>
+		<main class="main" id="main">
+			<nav>
+				<ul class="main_breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
```
Comment (A-Ostrovnyy) [inline submissions/f3d0t/html_movie_seat_booking/index.html]: For breadcrumbs better use `ol` tag.

File: submissions/f3d0t/html_movie_seat_booking/index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+							itemtype="https://schema.org/WebPage"
+							itemprop="item"
+							itemid="https://example.com/cinema/"
+							href="https://example.com/cinema/"
+						>
+							<span itemprop="name">Cinema</span></a
+						>
+						<meta itemprop="position" content="1" />
+					</li>
+					<li>›</li>
```
Comment (A-Ostrovnyy) [inline submissions/f3d0t/html_movie_seat_booking/index.html]: Use pseudoelements for things like this.

File: submissions/f3d0t/html_movie_seat_booking/index.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+					<li class="main_breadcrumbs__item" aria-current="page" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
+						<span itemprop="name">Fight Club</span>
+						<meta itemprop="position" content="3" />
+					</li>
+				</ul>
+			</nav>
+			<article class="movie">
+				<div class="movie__info">
+					<h1 class="movie__title">"Fight club" movie tickets booking</h1>
+					<h2 class="movie__time">Session time & date: <time datetime="2021-02-14 13:45">14.02.21 / 13:45</time></h2>
```
Comment (A-Ostrovnyy) [inline submissions/f3d0t/html_movie_seat_booking/index.html:62]: What the difference between this line and 66, that you choose heading `h2` there?

File: submissions/f3d0t/html_movie_seat_booking/index.html [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+					<li>›</li>
+					<li class="main_breadcrumbs__item" aria-current="page" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
+						<span itemprop="name">Fight Club</span>
+						<meta itemprop="position" content="3" />
+					</li>
+				</ul>
+			</nav>
+			<article class="movie">
+				<div class="movie__info">
+					<h1 class="movie__title">"Fight club" movie tickets booking</h1>
```
Comment (A-Ostrovnyy) [inline submissions/f3d0t/html_movie_seat_booking/index.html:61]: 🔥 very good movie)

File: submissions/f3d0t/html_movie_seat_booking/index.html [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+						<label class="form__seat" for="2-5"></label>
+						<input id="2-6" type="checkbox" class="visually-hidden form__seat_input" data-seat="6" data-row="2" value="2-6" disabled />
+						<label class="form__seat" for="2-6"></label>
+						<input id="2-7" type="checkbox" class="visually-hidden form__seat_input" data-seat="7" data-row="2" value="2-7" disabled />
+						<label class="form__seat" for="2-7"></label>
+						<input id="2-8" type="checkbox" class="visually-hidden form__seat_input" data-seat="8" data-row="2" value="2-8" />
+						<label class="form__seat" for="2-8"></label>
+						<input id="3-1" type="checkbox" class="visually-hidden form__seat_input" data-seat="1" data-row="3" value="3-1" />
+						<label class="form__seat" for="3-1"></label>
+						<label class="form__seat form__seat--placeholder"></label>
```
Comment (A-Ostrovnyy) [inline submissions/f3d0t/html_movie_seat_booking/index.html]: Avoid using empty elements for defining space. Use CSS grid for it.

File: submissions/f3d0t/html_movie_seat_booking/index.html [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+		</main>
+		<footer class="footer">
+			<address itemprop="address" itemscope="" itemtype="http://schema.org/PostalAddress">
+				<span>Cinema address:</span>
+				<strong>
+					<a href="https://goo.gl/maps/RmDcsY9GNsBvV5Zf7" target="_blank" itemprop="streetAddress" class="footer__link"
+						>50142, Italy, Via Annibal Caro, 71-1, Firenze FI</a
+					>
+				</strong>
+			</address>
```
Comment (A-Ostrovnyy) [inline submissions/f3d0t/html_movie_seat_booking/index.html:181]: 🔥

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below!

File: submissions/f3d0t/html_movie_seat_booking/index.html (hunk 3 above)
Comment (f3d0t) [reply submissions/f3d0t/html_movie_seat_booking/index.html:62]: because this heading is more informative, and It will be unique

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**